### ISSUE TITLE
Store the flow version somewhere in the definition

### ISSUE THREAD

**Issue by ericnewcomer** [nyaruka/floweditor#719]
It'd be nice when looking at old revisions to know which version of the editor created them.

**Comment by rowanseymour** [comment]
editor or flow version?

**Comment by ericnewcomer** [comment]
This is to know which version of the editor was used to create that definition. Sorry, fixing the title.

**Comment by rowanseymour** [comment]
could be a key `_ui` - the editor owns that and engine ignores it

**Comment by ericnewcomer** [comment]
Yup, agreed, somewhere under `_ui` is the logical place for it. Work here is mostly to figure out how the editor knows what version it is. Versioning command needs to write it to a config somewhere.

**Comment by ericnewcomer** [comment]
Of course, somebody handing a definition with a version isn't likely enough without the revision history. I suppose if we are good about patching editor bugs with minor flow migrations then that is less of an issue.
